EnglishFilter/sections/css_extended.txt

```diff
@@ -487,6 +487,10 @@ alluc.ee###resultlist > div#resultitems[-ext-has="> div"]~iframe:not([src])
 winaero.com#?#aside:has(.adsbygoogle)
 moozpussy.com##article.box > blockquote:has(> a[href][target="_blank"])
 forbes.com##.article-container div:has(> div > .ntv-ad)
+ekstrabladet.dk#?#.section-area:has(> .fnSpaceManagement > div > a[href^="https://side6.dk"])
+ekstrabladet.dk#?#.fnSpaceManagement:has(> div > div > div > .drfront > .df-articles > .df-container > .df-container-inner > .df-article > .df-article-content > .df-img-container > div > a[href^="https://shero.dk"])
+ekstrabladet.dk#?#.fnSpaceManagement:has(> .line > ul > li > a[href^="https://shero.dk"])
```

Comment:
Not shown in code for me. Also please move all rules to foreign.

Reply:
1, 2, 3 - not ads